shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createLevelDBDir:(NSString*) path
+{
+  NSFileManager* fm = [NSFileManager defaultManager];
+  // Setting NSFileProtectionCompleteUnlessOpen makes the directory accessible as long as the user has
+  // opened it once unlocked. The files are still stored on the disk encrypted (note for the chat database, it
+  // means we are encrypting it twice), and are inaccessible otherwise.
+  NSDictionary* noProt = [NSDictionary dictionaryWithObject:NSFileProtectionCompleteUnlessOpen forKey:NSFileProtectionKey];
```

Comment:
Here is a link to the docs which discusses this option https://developer.apple.com/reference/foundation/nsfileprotectioncompleteunlessopen